evdns.c

```diff
@@ -3562,6 +4222,14 @@ evdns_base_set_option_impl(struct evdns_base *base,
 		if (!(flags & DNS_OPTION_MISC)) return 0;
 		log(EVDNS_LOG_DEBUG, "Setting SO_SNDBUF to %s", val);
 		base->so_sndbuf = buf;
+	} else if (!strcmp(option, "use-vc")) {
+		if (!(flags & DNS_OPTION_MISC)) return 0;
+		log(EVDNS_LOG_DEBUG, "Setting use-vc option");
+		base->global_tcp_flags |= DNS_QUERY_USEVC;
```

Comment:
let's also add a check that `val == `NULL` and if not return an error, this will avoid questions about `evdns_base_set_option(base, "use-vc", "0")` (same for `ignore-tc`)

Reply:
`if (val && strlen(val)) return -1;`

ok

Reply:
I've rewritten this part. The problem with the original code is that it didn't work properly when I tried to parse `resolv.conf` file. When we have string like :
`options use-vc` val is actually == `""` (not NULL). So I had to retwrite it. Now parsing logic is rather permissive. 
All strings below are valid
```
options use-vc
options use-vc:
```
Invalid strings:
```
options use-vc:1
options use-vc:foo
```
Also variants `evdns_base_set_option` calls:
```
evdns_base_set_option(base, "use-vc", NULL);  // valid
evdns_base_set_option(base, "use-vc", "");  // valid
evdns_base_set_option(base, "use-vc", "1");  // invalid
```